operator/gitops/argocd/tekton-chains/base/chains-controller-deployment.yaml

```diff
@@ -14,6 +14,25 @@ spec:
             # Mount the secrets volume
             - mountPath: /etc/ssl/certs
               name: chains-ca-cert
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+              - "ALL"
+            # User 65532 is the nonroot user ID
+            runAsUser: 65532
```

Comment:
Do we need to run as specific user ID ?
isn't `runAsNonRoot`  enough ?

Reply:
Hey, the rationale was to define an explicit userID. Although I'm aware that the userID defaults to the UID defined in the image metadata but since we have defined `runAsNonRoot`, it won't default to root UID anyway.